### DISCUSSION TITLE
May Release - App overview section, new automation test mode, and a more performant Budibase

### DISCUSSION THREAD

**Discussion by joebudi** [Budibase/budibase#6157] [Announcements]
With the May release, we improved the stability of Budibase, introduced a number of user experience improvements, and fixed A LOT of bugs. Ultimately, you’re Budibase experience should be a lot smoother now!

---

### 🎨 Design feedback
Behind the scenes, we’re working on a number of substantial design improvements. If you’re interested in beta testing the new design changes, please [complete this form](https://bb.budibase.app/app/designbetatest#/beta-users/new/row).

---

### App Overview Section
The app list on the portal was becoming a little cluttered, so we did some tidying! The new app overview section is an all-new central control panel for your apps. It allows you to view your app’s status when it was last edited, and its version. This will be the main way to configure your app settings going forward, such as access control, backups, and much more.
![168029999-023c61c6-bcca-4f12-8f33-61102151187c](https://user-images.githubusercontent.com/49767913/171403261-74823e5d-c093-4531-a6e9-a3a39fbb5808.png)

### New Automation Test Mode
The new automation test mode sidebar makes testing your automations easier than ever before. Test your automations and view the results in the sidebar, so that you can continue tweaking your automation until it works the way you want!
<img width="1399" alt="169827402-2e35ecfd-2080-42e8-bc87-5e65fe7c430c" src="https://user-images.githubusercontent.com/49767913/171403235-24793d26-bf0e-44bc-a7f0-30253ff62679.png">

### Default Home Screen for All New Apps
Having to create a home screen for every new Budibase app was a bit of a pain. We’ve automatically added this, so you can just jump straight into the builder and start building your new app.

### Bug Fixes/Quality of Life Improvements
Loop (forEach) does not iterate on array #5751 
Remove row selection option for custom queries [#5491](https://github.com/Budibase/budibase/issues/5491)
Relationship options are empty after initial table creation [#5444](https://github.com/Budibase/budibase/issues/5444)
Improve Handlebars/JS UI for bindings in REST Connector Dynamic Variables [#4736](https://github.com/Budibase/budibase/issues/4736)
Make 'use bindings' persist in automation [#5910](https://github.com/Budibase/budibase/issues/5910)
Conditional required constraints are not properly cleared on condition change [#5341](https://github.com/Budibase/budibase/issues/5341)
Allow Budibase object store bucket names to be configurable through environment variables
[#5487](https://github.com/Budibase/budibase/issues/5487)
Trying to duplicate an auto-generated screen throws error [#5809](https://github.com/Budibase/budibase/issues/5809)
Automations cannot be executed by Basic user [#5760](https://github.com/Budibase/budibase/issues/5760)
null fields crash _stats views and subsequent table reads [#5850](https://github.com/Budibase/budibase/issues/5850)
Pre-fill app setup modal [#5188](https://github.com/Budibase/budibase/issues/5188)
Add the RejectUnauthorized switch to MySQL DB Integration SSL [#5530](https://github.com/Budibase/budibase/issues/5530)
SQL boolean columns - Budibase reads `false` value as `undefined` [#4982](https://github.com/Budibase/budibase/issues/4982)
Webhook Schema and Trigger URLs do not include domain [#5156](https://github.com/Budibase/budibase/issues/5156)
Cannot set role from Admin to No Access [#5367](https://github.com/Budibase/budibase/issues/5367)

---

## Coming up in the June Release 🤫
- [x] Snowflake integration
- [ ] User groups _[premium]_
- [x] Automation run logs
- [x] Single Docker image deployment
- [ ] SSO token support in REST connector

---

### Thank you to our community 🙌
Thank you to our community. We are ever so thankful for your feedback and assistance. We've recently surpassed 13,400 Github stars which is a great triumph. If you have not starred us, please do - it means a lot. Cheers!

**Comment by SantoshSrinivas79** [comment]
Great! Hope `{{ Current User}}` binding mentioned [here](https://github.com/Budibase/budibase/issues/4139#issuecomment-1122048116) is part of the SSO token support in REST connector work in the June release
